README.md

```diff
@@ -1,2 +1,24 @@
-# SimpleToDo
-This is a simple React to-do application where I used typescript for the first time. You can write a note about your task.
+![alt hydro1](simpletodo.png)
+
+# Simple ToDo Application
+- This is a React application where I used typescript and material UI for the first time.
+- User can add tasks and delete them when they are finished.
+- My biggest challenge: using CSS with material UI.
+
+## Features
+- The background color of the task cards depends on the urgency.
+- Application stores tasks in localstorage.
+
+## Technology
+
+- React
+- CSS
+- Typescript
+- Material UI
+
+## Getting Started
+
+- Clone the repository using the command git clone `https://github.com/kovesdieszter/SimpleToDo.git`.
+- Install all necessary packages asked by your IDE.
+- Run `npm start` from `simpletodo` directory.
```

Comment:
Fölösleges a projektnek még egy almappa, annak csak akkor van szerepe, ha több projektet is tárolsz egy repo-ban, ilyenkor mehet csak simán a gyökérbe.

Reply:
Ez a komplikált deploy miatt most kivételesen már így maradna.